src/frontend/src/app/App.offline.tsx

```diff
@@ -11,7 +11,10 @@ const discard = (error: any, _action: any, _retries: any) => {
   const { request, response } = error
   let message = 'Error, please contact support'
   if (response && response.error && typeof response.error === 'string') message = response.error
-  else if (response && typeof response === 'string') message = response
+  else if (response && typeof response === 'string')
+    message = response.includes('Token already exists')
+      ? 'It appears that a certificate has already been issued for this account'
+      : response
```

Comment:
@petarvujovic98 But actually I'll just go ahead and accept and merge it. We don't know that Academy is worth extra attention right now. But please respond anyway with your thoughts so that I know you understand what I'm saying.

Reply:
@ryancwalsh I definitely understand your thoughts, and feel the same way, I just thought we shouldn't spend too much time on this. As for the style/eslint configuration I also do not like this current setup and fell it's unreadable and error prone but for the time being I just went without setting anything up that would take too much time